### ISSUE TITLE
Analysis issue with common sentence: ご迷惑をおかけして申し訳ありませんでした。

### ISSUE THREAD

**Issue by rsimmons** [WorksApplications/SudachiDict#28]
(I hope this is the correct place to report this issue.)

Using the latest core dictionary, in any mode, there is this problem (it finds the word 決して):
```
$ echo ご迷惑をおかけして申し訳ありませんでした。 |  java -jar sudachi-0.5.2.jar -m B
ご	接頭辞,*,*,*,*,*	御
迷惑	名詞,普通名詞,サ変形状詞可能,*,*,*	迷惑
を	助詞,格助詞,*,*,*,*	を
おか	動詞,非自立可能,*,*,五段-カ行,未然形-一般	おく
けして	副詞,*,*,*,*,*	決して
申し訳	名詞,普通名詞,一般,*,*,*	申し訳
あり	動詞,非自立可能,*,*,五段-ラ行,連用形-一般	有る
ませ	助動詞,*,*,*,助動詞-マス,未然形-一般	ます
ん	助動詞,*,*,*,助動詞-ヌ,終止形-撥音便	ず
でし	助動詞,*,*,*,助動詞-デス,連用形-一般	です
た	助動詞,*,*,*,助動詞-タ,終止形-一般	た
。	補助記号,句点,*,*,*,*	。
EOS
```

but this works correctly:
```
$ echo おかけします |  java -jar sudachi-0.5.2.jar -m B
お	接頭辞,*,*,*,*,*	御
かけ	動詞,非自立可能,*,*,下一段-カ行,連用形-一般	掛ける
し	動詞,非自立可能,*,*,サ行変格,連用形-一般	為る
ます	助動詞,*,*,*,助動詞-マス,終止形-一般	ます
EOS
```

**Comment by kawahara-n** [comment]
Thank you for your report.
The issue is resolved and a fix will be made available in the next release.

**Comment by kazuma-t** [comment]
Fixed by #29
